test/sampling.jl

```diff
@@ -303,9 +303,17 @@ end
     @test p ≈ t
   end
 
-  grid = cartgrid(10, 10)
-  points = sample(grid, RegularSampling(100, 200))
-  @test length(collect(points)) == 20000
+  t = Triangle(cart(0, 0), cart(1, 0), cart(0, 1))
+  ps = sample(t, RegularSampling(10))
+  @test all(∈(t), ps)
+
+  t = Tetrahedron(cart(0, 0, 0), cart(1, 0, 0), cart(0, 1, 0), cart(0, 0, 1))
+  ps = sample(t, RegularSampling(10))
+  @test all(∈(t), ps)
```

Comment:
```suggestion
  @test all(∈(t), ps)
  @test length(collect(ps)) == 220
```
Same as above (`N * (N + 1) * (N + 2) / 6 == 220` for `N = 10`).